File: Process_Workshops/middle-letter/spec/middle_letter_spec.rb
require 'middle_letter'

describe "#get middle" do
  it "returns middle letter in hello" do
    string = "hello"
    expect(middle_letter(string)).to eq "l"
  end

  it "returns middle letter in goodbye" do
    string = "goodbye"
    expect(middle_letter(string)).to eq "d"
  end

  it "returns middle 2 chars of Arthur" do
    string = "Arthur"
    expect(middle_letter(string)).to eq "th"
  end
end

File: Process_Workshops/middle-letter-js/jasmine/spec/MiddleLetterSpec.js
describe("MiddleLetter", function() {
  it("returns single character for a single char length string", function() {
    expect(middleLetter("a")).toEqual("a")
  })

  it("returns both letters for a 2 character length string", function() {
    expect(middleLetter("ab")).toEqual("ab")
  })

  it("returns middle letter of 3 letter string", function() {
    expect(middleLetter("abc")).toEqual("b")
  })
})